src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -19,9 +19,15 @@ export interface AccordionProps {
   children: ReactNode;
   id: string;
   onToggleAll?: (open: boolean) => void;
+  analytics?: (accordionHeading: { id: string; title: string }) => void;
```

Comment:
Parameter should also be `accordionSection` or similar rather than `accordionHeading` (this is a misleading name)